### ISSUE TITLE
Suggestion

### ISSUE THREAD

**Issue by reaper666735** [fubaWoW/HandyNotes_WorldMapButton#5]
Hello again, 

so ty again for moving the icon position in 80300.03 :) 

i was simply wondering if we could have a small option in the handynotes options tab to change the 'Scale' of said icon ^^, 
it would help allot with keeping things tidy and in case we need to see something beneath it

ty.

**Comment by fubaWoW** [comment]
Yes i maybe will make it but i need to add a complete Options Panel for just Scaling and Positioning.
Currently have little time to sapre on this AddOn.

**Comment by Glurak18** [comment]
Is this stil lplanned i cant move the Button on the World Map and since blizz has his own map filter   this addon button is just under the blizzard one. :)
